nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -76,8 +76,7 @@ define([
         p = t('p'),
         blockquote = t('blockquote'),
         appStates = AppStates,
```

Comment:
Does this need to be renamed? `AppStates` can just be called directly, as it's not modified anywhere.

Reply:
And it's only used when initializing the FSM.